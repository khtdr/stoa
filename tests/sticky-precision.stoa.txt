0
0.0
0.00
12.46
12.46
