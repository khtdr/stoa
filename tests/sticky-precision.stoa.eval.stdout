0
0.0
0.00
12.25
122.46
122.46
